app/src/main/assets/navigation_route.json

```diff
@@ -0,0 +1 @@
2.606,"geometry_index":75,"location":[-118.291626,34.052403]},{"lanes":[{"indications":["left"],"valid":false,"active":false},{"indications":["straight"],"valid_indication":"straight","valid":true,"active":true},{"indications":["straight"],"valid_indication":"straight","valid":true,"active":true},{"indications":["right"],"valid":false,"active":false}],"location":[-118.291626,34.052519],"geometry_index":76,"admin_index":0,"weight":4.585,"is_urban":true,"mapbox_streets_v8":{"class":"primary"},"turn_duration":2.007,"turn_weight":2,"duration":4.117,"bearings":[0,180],"out":0,"in":1,"entry":[true,false]},{"entry":[false,true],"in":0,"bearings":[180,360],"duration":3.359,"turn_duration":0.007,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":1,"weight":4.106,"geometry_index":77,"location":[-118.291626,34.052672]},{"entry":[true,false],"in":1,"bearings":[0,180],"duration":0.907,"turn_duration":0.007,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":0,"weight":1.103,"geometry_index":78,"location":[-118.291628,34.052912]},{"entry":[false,true],"in":0,"bearings":[180,360],"duration":2.571,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":1,"weight":3.15,"geometry_index":79,"location":[-118.291628,34.052974]},{"entry":[true,false],"in":1,"bearings":[0,180],"duration":1.124,"turn_duration":0.007,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":0,"weight":1.369,"geometry_index":80,"location":[-118.291629,34.053154]},{"entry":[false,true],"in":0,"bearings":[180,360],"duration":4.407,"turn_duration":0.007,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":1,"weight":5.39,"geometry_index":81,"location":[-118.291629,34.053238]},{"entry":[false,true],"in":0,"bearings":[180,359],"duration":0.802,"turn_duration":0.019,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":1,"weight":0.959,"geometry_index":82,"location":[-118.29163,34.053535]},{"entry":[false,true],"in":0,"bearings":[179,360],"duration":8.827,"turn_duration":0.007,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":1,"weight":10.805,"geometry_index":83,"location":[-118.291631,34.053579]},{"entry":[false,true],"in":0,"bearings":[180,360],"duration":2.198,"turn_duration":0.007,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":1,"weight":2.684,"geometry_index":84,"location":[-118.291632,34.054021]},{"entry":[true,false],"in":1,"bearings":[0,180],"duration":1.55,"turn_duration":0.007,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":0,"weight":1.89,"geometry_index":85,"location":[-118.291633,34.054151]},{"entry":[false,true],"in":0,"bearings":[180,360],"duration":5.486,"mapbox_streets_v8":{"class":"primary"},"is_urban":true,"admin_index":0,"out":1,"weight":6.72,"geometry_index":86,"location":[-118.291633,34.054236]},{"entry":[true,false],"in":1,"bearings":[1,180],
```

Comment:
From where this source is coming?

Reply:
It's from our Directions API(from navigation SDK).